### PR TITLE
added image viewer example

### PR DESCRIPTION
**Pull request by penguindark**
**What's inside**
A new example for V.
This is a very simple image viewer that can accept:
- single files [JPEG, PNG, BMP, PSD, TGA, GIF (not animated), HDR, PIC, PNM]
- folders
- ziped folders
- a mix of the previous

**Image Operations:**
- Pan
- Zoom

### CODE REVIEW

File: examples/viewer/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# vviewer
+Image viewer for V
+
+This is an example of a simple image viewer written in V.
+
+## Usage
+
+the program can be invoked by the command line:
```
Comment (JalonSolov) [inline examples/viewer/README.md:8]: The

File: examples/viewer/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+file list: `viewer img1.jpg inmg2.bmp img3.tga`
+folder list: `viewer folder1 folder2`
+zip list: `viewer folder1.zip folder2.zip`
+
+All the folder/zips are scanned in order to find images
```
Comment (JalonSolov) [inline examples/viewer/README.md:14]: All folders/zips are scanned for images.

File: examples/viewer/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+The user can also mix the sources like files , folders and zips.
```
Comment (JalonSolov) [inline examples/viewer/README.md:15]: The user can mix files, folders, and zips.

File: examples/viewer/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+mixed list: `viewer img1.jpg img2.bmp folder1 folder2 img2.tga folder1.zip` 
+
+## Interactive usage
+
+run the viewer than drag and drop files,folders and zips on it.
```
Comment (JalonSolov) [inline examples/viewer/README.md:21]: **R**un the viewer th**e**n

File: examples/viewer/README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+## Accepted image format
+
+JPEG, PNG, BMP, PSD, TGA, GIF (not animated), HDR, PIC, PNM
+
+#### Functions
+The user can navigate through the files passed to the viewer.
+On each file can do the following operations:
```
Comment (JalonSolov) [inline examples/viewer/README.md:29]: The following operations can be performed on each image:

File: examples/viewer/README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+**cursor right**  - Next image
+**cursor  left**  - Previous image
+**cursor  up**    - Next folder
+**cursor  down**  - Previous folder
+**F** - Toggle full screen
+**R** - Rotate image of 90 degree
+**I** - Toggle the info text
+
+**mouse wheel** - next/previous images
+keep pressed **left  Mouse button** - Pan on the image
```
Comment (JalonSolov) [inline examples/viewer/README.md:49]: "keep pressed" -> "Hold"

File: examples/viewer/README.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+keep pressed **right Mouse button** - Zoom on the image
```
Comment (JalonSolov) [inline examples/viewer/README.md:50]: "keep pressed" -> "Hold"

**Comment by penguindark** [comment]
@JalonSolov Tnx for the review, I'll modify as the checks will pass.

**Comment by Larpon** [comment]
Awesome work!

**Comment by penguindark** [comment]
@spytheman why the tcc check is failing?
I tested it on my PC under windows and linux with tcc,msvc,gcc and compiel well on all.

**Pull request merged**